packages/connector-types/tsconfig.base.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@silverhand/ts-config/tsconfig.base",
+  "compilerOptions": {
+    "outDir": "build",
```

Comment:
this is more like a lib, so will it be better to keep the default option "lib" here?